scene/3d/area_3d.cpp

```diff
@@ -105,6 +105,56 @@ real_t Area3D::get_priority() const {
 	return priority;
 }
 
+void Area3D::set_wind_force_magnitude(real_t p_wind_force_magnitude) {
+	wind_force_magnitude = p_wind_force_magnitude;
+	if (is_inside_tree()) {
+		_initialize_wind();
+	}
+}
+
+real_t Area3D::get_wind_force_magnitude() const {
+	return wind_force_magnitude;
+}
+
+void Area3D::set_wind_attenuation_factor(real_t p_wind_force_attenuation_factor) {
+	wind_attenuation_factor = p_wind_force_attenuation_factor;
+	if (is_inside_tree()) {
+		_initialize_wind();
+	}
+}
+
+real_t Area3D::get_wind_attenuation_factor() const {
+	return wind_attenuation_factor;
+}
+
+void Area3D::set_wind_source_path(const NodePath &p_wind_source_path) {
+	wind_source_path = p_wind_source_path;
+	if (is_inside_tree()) {
+		_initialize_wind();
+	}
+}
+
+const NodePath &Area3D::get_wind_source_path() const {
+	return wind_source_path;
+}
+
+void Area3D::_initialize_wind() {
+	Node3D *p_wind_source = Object::cast_to<Node3D>(get_node(wind_source_path));
+	ERR_FAIL_NULL(p_wind_source);
+	Transform3D global_transform = p_wind_source->get_transform();
+	Vector3 wind_direction = -global_transform.basis.get_axis(Vector3::AXIS_Z).normalized();
+	Vector3 wind_source = global_transform.origin;
+
+	// If Node3D isn't available, negate wind by setting magnitude to zero.
+	real_t temp_magnitude = wind_source_path.is_empty() ? 0.0 : wind_force_magnitude;
```

Comment:
This check should be at the beginning of the method, otherwise `get_node(wind_source_path)` will fail anyway and return early.